helm/charts/hpe-csi-driver/README.md

```diff
@@ -43,25 +42,57 @@ It's recommended to create a [values.yaml](https://github.com/hpe-storage/co-dep
 ### Installing the Chart
 To install the chart with the name `hpe-csi`:
 
+Add HPE helm repo:
 ```
 helm repo add hpe https://hpe-storage.github.io/co-deployments
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml
+helm repo update
+helm search repo hpe-csi-driver
```

Comment:
Why would you do this? (helm search)

Reply:
this is to find the available versions to install.